package.json

```diff
@@ -50,6 +50,9 @@
         "build:prod:npm": "BABEL_ENV=production OUTPUT=dist LANGUAGE=en-US REACT=true yarn build:prod:dist",
         "clean": "rm -rf dist lib i18n/json i18n/*.js reports styleguide",
         "copy:styles": "cpx './src/**/*.scss' lib",
+        "cy:open": "yarn cy:wait; cypress open --project ./test",
+        "cy:run": "yarn cy:wait; cypress run --project ./test --spec test/cypress/**/*.spec.js",
+        "cy:wait": "wait-on tcp:6060",
```

Comment:
Are there any concerns with hardcoding the port? Parallelism issues in Travis maybe?